src/components/Signup/Signup.js

```diff
@@ -150,9 +154,9 @@ const Signup = function ({ location, dependencies: { dopplerLegacyClient, origin
             />
           ),
         });
+        setSubmitting(false);
       }
     } finally {
-      setSubmitting(false);
     }
```

Comment:
Por que se cambian los llamados a `setSubmitting`?

Reply:
En el caso que la llamada a registrar da ok, se redirige a la pagina de confirmacion y entonces este setSubmitting se quiere ejecutar sobre un componente desmontado.  Ese es el error que daba el test.

Reply:
Buenisimo, gracias por la aclaración!

Creo que deberías quitar todo el bloque finally.

Reply:
![image](https://user-images.githubusercontent.com/2439363/106294381-dca84e00-622d-11eb-8865-341ac3ba3ee9.png)

Reply:
Esto ocurre solamente en el form de signup pq hay un test que valida la página de confirmación luego del submit, en otros formularios no tenemos este tipo de test.

Reply:
> Buenisimo, gracias por la aclaración!
> 
> Creo que deberías quitar todo el bloque finally.

No me permite tener un try sin un finally 🤔 Me da error en tiempo de ejecucion directamente. 
![image](https://user-images.githubusercontent.com/2439363/106296666-ba63ff80-6230-11eb-9218-9ece3b01e63e.png)

Reply:
I've removed the try.